File: get_f473.py
#!/usr/bin/env python3
import sqlite3

conn = sqlite3.connect('features.db')
cur = conn.cursor()
cur.execute('SELECT id, priority, category, name, description, steps FROM features WHERE id = 473')
row = cur.fetchone()

if row:
    print(f'ID: {row[0]}')
    print(f'Priority: {row[1]}')
    print(f'Category: {row[2]}')
    print(f'Name: {row[3]}')
    print(f'Description: {row[4]}')
    print(f'Steps: {row[5]}')
else:
    print('Feature #473 not found')

conn.close()

File: get_f473.js
const sqlite3 = require('sqlite3');
const db = new sqlite3.Database('./features.db');

db.get('SELECT id, priority, category, name, description, steps FROM features WHERE id = 473', (err, row) => {
  if (err) {
    console.error('Error:', err);
  } else if (row) {
    console.log('ID:', row.id);
    console.log('Priority:', row.priority);
    console.log('Category:', row.category);
    console.log('Name:', row.name);
    console.log('Description:', row.description);
    console.log('Steps:', row.steps);
  } else {
    console.log('Feature #473 not found');
  }
  db.close();
});
